e2e/data/smoke/rook_api_external.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
a constant port would be fine, I'm just suggesting that we load the yaml from `demo/kubernetes` and then replace the const during the test, rather than copying the whole yaml. yes, for another day...